doc/source/general/compiling.rst

```diff
@@ -18,7 +18,33 @@ When you try to run your script, the first thing that kOS does is transform your
 
 This process of transforming your script into Machine Language, or "ML" is called "Compiling".
 
-The ":ref:`RUN`" and ":ref:`RUNPATH`" commands do this silently, without telling you. This is why you may have noticed the universe slightly stutter for a moment when you first run your program. Compiling is hard work for the universe to do.
+The :key:`RUN` and :func:`RUNPATH` commands do this silently, without telling
+you.  The :key:`COMPILE` command explicitly compiles the file and saves it for
+future use.
+
```

Comment:
Meh, Sphinx is already complaining about a broken link in the GUI documentation, so that shouldn't hold us up.  If we're in that file anyways making changes, we might as well add the label.